test/objects.js

```diff
@@ -609,12 +609,12 @@
       var view2 = new Uint8Array(shared.buffer, 2, 2);
       assert.notOk(_.isEqual(view1, view2), 'same buffer with different offset is not equal');
 
-      // Some older browsers support typed arrays but not DataView.
-      if (typeof DataView !== 'undefined') {
-        assert.ok(_.isEqual(new DataView(u8.buffer), new DataView(u8b.buffer)), 'Identical DataViews are equal');
-        assert.ok(_.isEqual(new DataView(u8.buffer), new DataView(i8.buffer)), 'Identical DataViews of different typed arrays are equal');
-        assert.notOk(_.isEqual(new DataView(u8.buffer), new DataView(u16.buffer)), 'Different DataViews with different length are not equal');
-        assert.notOk(_.isEqual(new DataView(u8.buffer), new DataView(u16one.buffer)), 'Different DataViews with different byte data are not equal');
+      var DataViewImpl = typeof NativeDataView !== 'undefined' ? NativeDataView : typeof DataView !== 'undefined' ? DataView : undefined; // Browser could override DataView
```

Comment:
Please turn the new comment into a full sentence and put it on a separate line above the code it annotates. That keeps it consistent with the literate style of the library.